lib/index.js

```diff
@@ -93,17 +91,17 @@ internals.registrationConfig = (options, rootServer) => {
             config.models = require(config.models);
         }
         else {
-            const relativeTo = rootServer.realm.settings.files.relativeTo || '';
+            const relativeTo = rootServer.settings.files.relativeTo || '';
```

Comment:
The thing that should be deleted should probably be `rootServer` rather than `realm`– a server does not have `settings` but a realm does.

Reply:
ah, doi. yea, I wassuper lazy on updating variable names to match what they actually describe now. Thanks for pointing that out, homie! 🙏